build/tutorials/tutorials-contest/evoting-avalanche-dapp/README.md

```diff
@@ -0,0 +1,635 @@
+# Create a Voting dApp on Avalanche using ReactJS
+
+## Introduction
```

Comment:
[Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022)